src/Illuminate/Console/Application.php

```diff
@@ -55,11 +55,13 @@ public function __construct(Container $laravel, Dispatcher $events, $version)
      */
     public function call($command, array $parameters = [])
     {
-        $parameters['command'] = $command;
+        $parameters = collect($parameters)->prepend($command);
 
         $this->lastOutput = new BufferedOutput;
 
-        return $this->find($command)->run(new ArrayInput($parameters), $this->lastOutput);
+        $this->setCatchExceptions(false);
```

Comment:
I wasn't arguing that. I'm saying after execution is done, that value needs restoring to what it was before it was set to false.